Orange/data/variable.py

```diff
@@ -887,114 +639,106 @@ class TimeVariable(ContinuousVariable):
              ')$')
     _matches_iso_format = re.compile(REGEX).match
 
-    # UTC offset and associated timezone. If parsed datetime values provide an
-    # offset, it is used for display. If not all values have the same offset,
-    # +0000 (=UTC) timezone is used and utc_offset is set to False.
-    utc_offset = None
-    timezone = timezone.utc
-
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.have_date = 0
-        self.have_time = 0
```

Comment:
What equivalent to `TimeVariable.{have_date,have_time}` do we have now? Do all datetimes just always show a date/time component, regardless of if the data contained it? Could `format` inferred in `TimeVariable.column_to_datetime()` below be saved for future use? It doesn't look so nice if yearly data is displayed as full datetime '1960-01-01 00:00:00+00:00'.